### PR TITLE
Avalanche Stats

### PR DESCRIPTION
**Pull request by 0fatih**
I saw on twitter that avalanche has a stats website. I think we should include this in the docs.

### CODE REVIEW

File: build/tutorials/nodes-and-staking/README.md [hunk 1]
```diff
@@ -11,4 +11,4 @@
 | [**Set up Node Monitoring**](setting-up-node-monitoring.md) | Set up infrastructure to monitor an instance of AvalancheGo. |
 | [**Stake AVAX, by Validating or Delegating, with the Avalanche Wallet**](staking-avax-by-validating-or-delegating-with-the-avalanche-wallet.md) | Stake AVAX from the Avalanche wallet. |
 | **​**[**Upgrade Your AvalancheGo Node**](https://docs.avax.network/build/tutorials/nodes-and-staking/upgrade-your-avalanchego-node)**​** | Upgrade your Avalanche node. |
-
+| **​**[**Avalanche Stats**](https://docs.avax.network/build/tutorials/nodes-and-staking/upgrade-your-avalanchego-node)**​** | Avalanche Stats. |
```
[the remark this answers is not in the document]
Comment (cgcardona) [reply build/tutorials/nodes-and-staking/README.md:14]: Also add the link to `SUMMARY.md`

**Review by cgcardona — CHANGES_REQUESTED** [review]
Thanks. Requested changes.

**Comment by 0fatih** [comment]
I made the changes you requested. Please let me know if I have another mistake

File: build/tools/avalanche-stats.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+Minumum gas price for the last 2000 block.
+
+### Minumum Gas Price (Last 8 hours)
+
+Minumum gas price for the last 8 hours.
+
+### Daily Fees Burned
+
+Shows how much gas is burned in the last day.
```
Comment (yulin-dong) [inline build/tools/avalanche-stats.md:123]:
```suggestion
Shows how much gas is burned in every day.
```

File: build/tools/avalanche-stats.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+### Daily Gas Used
+
+Shows how much gas is used in the last day.
```
Comment (yulin-dong) [inline build/tools/avalanche-stats.md:127]:
```suggestion
Shows how much gas is used in every day.
```

File: build/tools/avalanche-stats.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+### Daily Tx Count
+
+Shows how many transaction were made in the last day.
```
Comment (yulin-dong) [inline build/tools/avalanche-stats.md:131]:
```suggestion
Shows how many transaction were made in every day.
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Comment by 0fatih** [comment]
@yulin-dong thanks!

File: build/tools/README.md [hunk 5]
```diff
@@ -12,3 +12,4 @@
 
 [**Postman collection**](postman-avalanche-collection.md): An AvalancheGo API template and environment collection for [Postman](https://postman.com/), a tool to simplify API development and experimentation. The collection covers all of AvalancheGo APIs, so you can easily issue commands to a node and examine the output in a nicely formatted, readable way.
 
+[**Avalanche Stats**](avalanche-stats.md): You can learn the status of Avalanche and nodes.
```
Comment (yulin-dong) [inline build/tools/README.md:15]:
```suggestion
[**Avalanche Stats**](avalanche-stats.md): Dashboard for Avalanche stats.
```

File: build/tools/avalanche-stats.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
+# Avalanche Stats
+
+## Introduction
+
+We created a [stats][https://stats.avax.network/] website that you can learn a lot of things about avalanche's current status.
```
Comment (yulin-dong) [inline build/tools/avalanche-stats.md:5]:
```suggestion
We have created a [stats][https://stats.avax.network/] website from where you can learn a lot about avalanche's current stats.
```

File: build/tools/avalanche-stats.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+
+## What is in there
```
Comment (yulin-dong) [inline build/tools/avalanche-stats.md:7]:
```suggestion
## What are included?
```

File: build/tools/avalanche-stats.md [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+
+We have 3 main concepts in our stats: Staking, Validator Health Check and C-Chain Activity.
```
Comment (yulin-dong) [inline build/tools/avalanche-stats.md:9]:
```suggestion
We have 3 main areas in our stats: Staking, Validator Health Check and C-Chain Activity.
```

File: build/tools/avalanche-stats.md [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+
+## Staking
+
+You can access this part from [this][https://stats.avax.network/dashboard/staking/] link. And you can see:
```
Comment (yulin-dong) [inline build/tools/avalanche-stats.md:13]:
```suggestion
You can access this part from [this][https://stats.avax.network/dashboard/staking/] link from which you will get following info:
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Review by cgcardona — APPROVED** [review]
Thanks 🎩

**Pull request merged**